### PR TITLE
Add lines to flight map

### LINKED ISSUE #148: Great circle line on maps
**Is your feature request related to a problem? Please describe.**
A map with two points and a large gap between with nothing else plotted looks empty.

**Describe the solution you'd like**
A great circle line could be plotted on the map between the two airports.

**Describe alternatives you've considered**
Plotting the route would require much more additional data and processing, this is a simpler solution that still gets the job done.

### PR DESCRIPTION
**Pull request by calumshep**
- Added great circle line between map markers (closes #148)
- Route from flights dispatched via SimBrief now archived and displayed in logbook

### CODE REVIEW

File: database/migrations/2020_12_29_135542_add_route_to_archived_flights_table.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('archived_flights', function (Blueprint $table) {
+            //
```
Comment (hcphoon01) [inline database/migrations/2020_12_29_135542_add_route_to_archived_flights_table.php]: Include something in the down method, for example `$table->dropColumn('route');`

File: resources/views/flights/show.blade.php [hunk 2]
```diff
@@ -419,80 +417,95 @@ class="modal fade"
...
-                            'text': curr.icao + ' - ' + curr.name,
-                            'disabled': false
+            },
+            locale: {
+                emptyTitle: 'Start typing to search...',
+                statusInitialized: '',
+            },
+            preprocessData: function(data)
+            {
+                var aircraft = [];
```
Comment (hcphoon01) [inline resources/views/flights/show.blade.php]: Change this to a `let` and also change the name from `aircraft` to something like `result`

**Review by hcphoon01 — DISMISSED** [review]

File: database/migrations/2020_12_29_135542_add_route_to_archived_flights_table.php (hunk 1 above)
Comment (calumshep) [reply database/migrations/2020_12_29_135542_add_route_to_archived_flights_table.php]: Resolved in 89edd33fbb72c544ea4e78e852b76e1f0427a456

File: resources/views/flights/show.blade.php (hunk 2 above)
Comment (calumshep) [reply resources/views/flights/show.blade.php]: Resolved in c2d8a63d5e3083ab574bae6541d6b3f3c49f0e19

**Review by hcphoon01 — APPROVED** [review]

**Pull request merged**